lms/djangoapps/commerce/api/v1/urls.py

```diff
@@ -4,15 +4,19 @@
 
 from commerce.api.v1 import views
 
-
 COURSE_URLS = patterns(
     '',
     url(r'^$', views.CourseListView.as_view(), name='list'),
     url(r'^{}/$'.format(settings.COURSE_ID_PATTERN), views.CourseRetrieveUpdateView.as_view(), name='retrieve_update'),
 )
 
+ORDER_URLS = patterns(
+    '',
+    url(r'^(?P<number>[-\w]+)/$', views.OrderView.as_view(), name='detail'),
+)
+
 urlpatterns = patterns(
     '',
     url(r'^courses/', include(COURSE_URLS, namespace='courses')),
-
+    url(r'^orders/', include(ORDER_URLS, namespace='orders')),
```

Comment:
We don't need arch council to review a proxy of for an API that's already been reviewed.